docs/Developers/Development_Setup.rst

```diff
@@ -39,7 +39,13 @@ master branch from and all of its dependencies with pip using
 
 ::
 
-    (venv)$ pip3 install coala-bears --pre
+    (venv)$ git clone https://github.com/coala/coala
+    (venv)$ cd coala
+    (venv)$ pip3 install .
+    (venv)$ cd -
+    (venv)$ git clone https://github.com/coala/coala-bears
+    (venv)$ cd coala-bears
+    (venv)$ pip3 install .
```

Comment:
IMO the paragraph below should be modified as well, explaining to use `-e` at the previous instal commands, you don't want to install it uneditable and then editable, thats not too good for your virtualenv